pyuploadcare/conf.py

```diff
@@ -35,6 +35,8 @@
 
 user_agent_extension = None
 
-batch_chunk_size = 500
+batch_chunk_size = 100
+# because of API docs: https://uploadcare.com/api-refs/rest-api/v0.7.0/#operation/filesStoring
```

Comment:
We don't need this in the code. This is a part of changelog/meta info.
Probably as a part of fixes, as in all API versions documented limit is 100
```suggestion

batch_chunk_size = 100
```

Reply:
Agree! so I add `Fixed`part into `HISTORY.md` and move link into it